x-pack/plugins/maps/public/connected_components/toolbar_overlay/set_view_control/set_view_control.tsx

```diff
@@ -215,3 +664,64 @@ export class SetViewControl extends Component<Props, State> {
     );
   }
 }
+
+function convertLatLonToUTM(lat: any, lon: any) {
```

Comment:
[`usng.LLtoUTM` appears to accept both number or string](https://github.com/codice/usng.js/blob/master/src/usng.ts#L408-L409).

```suggestion
function convertLatLonToUTM(lat: number | string, lon: number | string) {
```